Formula/ghc.rb

```diff
@@ -17,7 +17,8 @@ class Ghc < Formula
   end
 
   depends_on "python@3.9" => :build
-  depends_on "sphinx-doc" => :build
+  depends_on "llvm" if Hardware::CPU.arm?
```

Comment:
Currently, this causes `llvm` to be pulled in as a runtime dependency.

Reply:
Pretty sure it should be. It should actually also still be using system Clang to compile.